packages/sdk/wallets/wallet-ledger/src/ledger-wallet.ts

```diff
@@ -68,22 +69,32 @@ export class LedgerWallet extends RemoteWallet<LedgerSigner> implements ReadOnly
     }
   }
 
-  signTransaction(txParams: CeloTx): Promise<EncodedTransaction> {
-    // CeloLedger does not support maxFeePerGas and maxPriorityFeePerGas yet
-    txParams.gasPrice = txParams.gasPrice ?? txParams.maxFeePerGas
-    if (txParams.maxFeePerGas || txParams.maxPriorityFeePerGas) {
-      console.info(
-        'maxFeePerGas and maxPriorityFeePerGas are not supported on Ledger yet. Automatically using gasPrice instead.'
+  async signTransaction(txParams: CeloTx): Promise<EncodedTransaction> {
+    if (txParams.gasPrice) {
+      if (txParams.feeCurrency && txParams.feeCurrency !== '0x') {
+        // NOTE: celo-legacy is not supported anymore
+        throw new Error(
+          'celo-legacy transactions are not supported anymore, please try sending a more modern transaction instead (eip1559, cip64, etc.)'
+        )
+      }
+      throw new Error(
+        'ethereum-legacy transactions are not supported anymore, please try sending a more modern transaction instead (eip1559, cip64, etc.)'
```

Comment:
technically i think the ledger app never supported them so more correct to say 

`ethereum-legacy transactions are not supported by the celo ledger app`